src/data/tilesets/graphics.h

```diff
@@ -1,6 +1,6 @@
 const u32 gTilesetTiles_Petalburg[] = INCBIN_U32("data/tilesets/secondary/petalburg/tiles.4bpp.lz");
 
-const u16 gTilesetPalettes_Petalburg[][16] =
+const u16 ALIGNED(4) gTilesetPalettes_Petalburg[][16] =
```

Comment:
Is there a reason these need to be aligned? If so, are we sure that we have the correct behaviour if you add a new tileset through PoryMap?

Reply:
I don't know the full convo (@SBird1337 might, as they're credited for the fix)
- https://github.com/aarant/pokeemerald/commit/782889d5334b4a7bf78449ce903fbb1f8c9ba2f7
- https://discord.com/channels/442462691542695948/442465020291317760/1045406595770765413

Reply:
Removed